server/app.py

```diff
@@ -160,6 +172,75 @@ def main():
     logger.info("Close connection with sc-server")
     client.disconnect()
 
+scs_paths = set()
+scs_exclude_paths = set()
+def search_knowledge_bases(root_path: str):
```

Comment:
Method name is a bit confusing - we have one common knowledge base, it's like a search for kb sources